components/application-connectivity-validator/internal/validationproxy/handler.go

```diff
@@ -295,7 +295,8 @@ func createReverseProxy(destinationHost string, reqOpts ...requestOption) *httpu
 		Director: func(request *http.Request) {
 			request.URL.Scheme = "http"
 			request.URL.Host = destinationHost
-
+			request.Host = ""
+			delete(request.Header, "X-Forwarded-Client-Cert")
```

Comment:
👍

Reply:
There is a method for that: https://golang.org/pkg/net/http/#Header.Del

Reply:
Changed